lib/uvobot/uvo_scraper.rb

```diff
@@ -24,14 +24,14 @@ def issue_ready?(release_date)
 
     def get_announcements(release_date)
       date = release_date.strftime('%d.%m.%Y')
-      search_query = { kcpv: IT_CONTRACTS_CODE, dzOd: date, dzDo: date }
-      html = @html_client.post(SEARCH_URL, search_query).body
+      code = IT_CONTRACTS_CODE.tr(' ', '+')
```

Comment:
Uz by sme to mohli premenovat na `IT_CONTRACTS_CODES` dat ako pole a tuto dat normalny join s +.